### DISCUSSION TITLE
Having Troubles with outdated links on my /sitemap.xml

### DISCUSSION THREAD

**Discussion by maxklema** [swellstores/community#18] [Help] [answered]
Hey,

I just deployed a site (https://www.microgardengreens.com). However, when I try to get the sitemap at /sitemap.xml, it seems swell is using the old domain (swell.store). How can I change this to the new domain to update my sitemap?

![help image](https://user-images.githubusercontent.com/80615123/205387660-03b0a54d-65b3-4188-a70c-6d367bd1c6c2.png)

Any help would be appreciated.

**Comment by logeshswell** [comment, marked as answer]
@agileinx I guess you wish to set up a custom domain (https://www.microgardengreens.com) for your swell store. If so, please follow the steps mentioned in this guide:
https://developers.swell.is/storefronts/custom-domains-and-ssl

I see you are 1 step away from finishing the setup and it is nothing but [verifying your custom domain](https://developers.swell.is/storefronts/custom-domains-and-ssl#verifying-the-domain) at Swell. Please keep us posted if you find any issues in domain verification.

**Reply by maxklema** [reply]
Hello, I am having trouble verifying my domain. I configured all the necessary CNAMEs in the DNS records but still failed verification. Can you help?

**Reply by maxklema** [reply]
@logeshswell 

![image](https://user-images.githubusercontent.com/80615123/205993108-5002f066-0ae1-402c-a937-4cc46d975a0d.png)

I was able to verify my domain, but why does verification only work when it is www.example.com instead of just example.com?

My sitemap is still using the swell.store extension, is there an ETA on when my sitemap will be updated?

**Reply by logeshswell** [reply]
Yes @agileinx, I guess there might be some restrictions with the use of subdomains in the custom domain verification.
I also noticed that you are redirecting `www.example.com` to `example.com` as a workaround, which is recommended for your usecase.
